rest/data/host_create.go

```diff
@@ -253,16 +256,44 @@ func (dc *DBCreateHostConnector) MakeIntentHost(taskID, userID, publicKey string
 	return cloud.NewIntent(d, d.GenerateName(), provider, options), nil
 }
 
+// GetLogs is used by the /host/{container_id}/logs route to retrieve the logs for the given container.
+func (dc *DBCreateHostConnector) GetLogs(ctx context.Context, containerId string, parent *host.Host) (*cloud.LogReader, error) {
+	settings, err := evergreen.GetConfig()
```

Comment:
Instead of doing this, can we pass the API version into the function? That way we don't need to pass around the entire object for 1 field, and also we avoid having this package doing queries

Reply:
Yeah i was torn over which organization was cleaner. I'll make the change.

Reply:
GetClient saves the entirety of settings, so it doesn't work to just pass in the API version. We could pass in settings itself instead of calling evergreen.GetConfig() here if you want to avoid this package having queries.